userd/userd.go

```diff
@@ -71,17 +50,29 @@ func (ud *Userd) Init() error {
 		return err
 	}
 
-	reply, err := ud.conn.RequestName(busName, dbus.NameFlagDoNotQueue)
-	if err != nil {
-		return err
+	ud.dbusIfaces = []dbusInterface{
+		&Launcher{},
 	}
+	for _, iface := range ud.dbusIfaces {
+		reply, err := ud.conn.RequestName(iface.Name(), dbus.NameFlagDoNotQueue)
```

Comment:
Is each interface going to be exposed as a separate DBus service? Before there was one service with multiple interfaces on a single path. With this change it's service per one of `dbusIfaces`, with multiple interfaces inside.

Reply:
I suspect this is the main point of the change. We don't want everything under .../Launcher, for example.